src/content/posts/new-wireguard-mode/index.md

```diff
@@ -0,0 +1,97 @@
+---
+title: "A more user-friendly transparent mode, based on WireGuard"
+summary: | 
+    The next release of mitmproxy is slated to ship with a new alternative to the existing "transparent" mode, based on
+    WireGuard. The new "wireguard" mode was designed to be much simpler to set up and use, and in most cases, it does not
+    require taking any actions with elevated privileges.
+date: 2022-09-23
+weight: 10
+tags: [tech]
+author:
+    name: Fabio Valentini
+    twitter: decathorpe
+---
+
+The next release of mitmproxy is slated to ship with a new alternative to the existing `transparent` mode, based on
+WireGuard. The new `wireguard` mode was designed to be much simpler to set up and use, and in most cases, it does not
+require taking any actions with elevated privileges.
+
+Connecting another device and routing its traffic through mitmproxy becomes as simple as connecting to a WireGuard VPN
+--- either by using the WireGuard configuration that is printed by mitmproxy, or by scanning the QR code that is shown
```

Comment:
```suggestion
--- either by using the WireGuard configuration file generated by mitmproxy, or by scanning the QR code that is shown
```

Reply:
I'll change this with the next push, thanks.